CleanSwift/CleanSwift/Astronomy/AstronomyModel.swift

```diff
@@ -0,0 +1,18 @@
+// AstronomyModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Модель экрана астрономии
+enum AstronomyModel {
```

Comment:
```suggestion
enum AstronomyModel {
```
раньше Женя писал коммент про слово Model  в модели, но если Саша сказал называть всем в одном стиле то видимо можно оставить)